### DISCUSSION TITLE
How to generate answers from a loaded GPT model

### DISCUSSION THREAD

**Discussion by bridgearchway** [hpcaitech/ColossalAI#2902] [Community | Q&A] [answered]
I've loaded my pretrained model. But when I was doing 'model.generate()' and 'tokenizer.batch_decode()', the output is just nonsense. Just get stuck here.

**Comment by ht-zhou** [comment, marked as answer]
Thanks for your feedback.The code we support for training in example is a demo.You need to get
1) pre-trained model params
2) high quality dataset
to train for over 1 million episodes to get an actor with good performance.
<img width="1028" alt="image" src="https://user-images.githubusercontent.com/70618399/222391968-e8122aba-75e3-4db6-b552-15f85cf6af45.png">
